pkgs/servers/blockbook/default.nix

```diff
@@ -45,7 +51,7 @@ buildGoPackage rec {
     description = "Trezor address/account balance backend";
     homepage = "https://github.com/trezor/blockbook";
     license = licenses.agpl3;
-    maintainers = with maintainers; [ mmahut ];
+    maintainers = with maintainers; [ mmahut maintainers."1000101" ];
```

Comment:
You can drop `maintainers.` before "1000101"

Reply:
Are you sure? https://github.com/NixOS/nixpkgs/pull/66842

Reply:
Not sure, but there are other places where it's being used like this:

```
nixos/tests/dokuwiki.nix:35:  meta.maintainers = with pkgs.lib.maintainers; [ "1000101" ];
nixos/tests/trezord.nix:4:    maintainers = [ mmahut "1000101" ];
nixos/tests/trickster.nix:4:    maintainers = [ "1000101" ];
```

So if that does not work, we should fix these.

Reply:
Yeap, I am going to fix them in a separate PR.

Reply:
Maybe we should only allow maintainer names that can be safely expressed as variables in nix.